### PR TITLE
fix: [STO-5660]: STO Subscriptions

### PR DESCRIPTION
**Pull request by ahokinson**
### Summary

STO subscription options will no longer be gated by active license.

#### Screenshots
CleanShot 2023-08-02 at 00 15 30
CleanShot 2023-08-02 at 00 14 36

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by bhavya-sinha — APPROVED** [review]

**Review by tan-nhu — DISMISSED** [review]

File: src/modules/27-platform/auth-settings/pages/subscriptions/SubscriptionsPage.tsx [hunk 1]
```diff
@@ -114,7 +114,7 @@ const SubscriptionsPage: React.FC = () => {
           FF_LICENSE_STATE === LICENSE_STATE_VALUES.ACTIVE && accumulator.push(card)
           break
         case ModuleName.STO:
-          licenseInformation['STO']?.status === LICENSE_STATE_VALUES.ACTIVE && accumulator.push(card)
+          accumulator.push(card)
```
Comment (atmollo-harness) [inline src/modules/27-platform/auth-settings/pages/subscriptions/SubscriptionsPage.tsx:117]: Do we want to show non active licenses in the UI?

File: src/modules/27-platform/auth-settings/pages/subscriptions/plans/Plan.module.scss [hunk 2]
```diff
@@ -92,6 +92,9 @@
 .chaosColor {
   color: var(--magenta-700) !important;
 }
+.stoColor {
+  color: var(--primary-7) !important;
```
Comment (chris-harness) [inline src/modules/27-platform/auth-settings/pages/subscriptions/plans/Plan.module.scss:96]: Is this a color that has been used for STO elsewhere?

File: src/modules/27-platform/auth-settings/pages/subscriptions/SubscriptionsPage.tsx (hunk 1 above)
Comment (chris-harness) [reply src/modules/27-platform/auth-settings/pages/subscriptions/SubscriptionsPage.tsx:117]: Seems like some other modules show inactive plans, so might be OK? Definitely some inconsistency across modules, which makes it confusing, but not much we can do about that.

**Review by chris-harness — COMMENTED** [review]
Generally looks good. Just one question.

File: src/modules/27-platform/auth-settings/pages/subscriptions/SubscriptionsPage.tsx (hunk 1 above)
Comment (ahokinson) [reply src/modules/27-platform/auth-settings/pages/subscriptions/SubscriptionsPage.tsx:117]: It's probably the only line being asked for. `STO-5660: Implement Subscriptions page (STO) for Accounts without STO licenses`

**Review by atmollo-harness — APPROVED** [review]

File: src/framework/LicenseStore/licenseStoreUtil.tsx [hunk 3]
```diff
@@ -44,6 +44,10 @@ export const defaultLicensesByModule: {
   },
   CV: {
     edition: Editions.FREE
+  },
+  // TODO: change to free when free plan is supported
+  STO: {
+    edition: Editions.ENTERPRISE
```
Comment (devinder-harness) [inline src/framework/LicenseStore/licenseStoreUtil.tsx:50]: are we giving away `ENTERPRISE` on purpose?

**Review by tan-nhu — APPROVED** [review]

**Pull request merged**